cmd/chainbridge/config.go

```diff
@@ -73,14 +73,18 @@ func (c *Config) validate() error {
 		if chain.Type == "" {
 			return fmt.Errorf("required field chain.Type empty for chain %d", chain.Id)
 		}
-
 		if chain.Endpoint == "" {
 			return fmt.Errorf("required field chain.Endpoint empty for chain %d", chain.Id)
 		}
-
 		if chain.Name == "" {
 			return fmt.Errorf("required field chain.Name empty for chain %d", chain.Id)
 		}
+		if chain.Id == 0 {
```

Comment:
@ansermino are we never going to have a chain with `id` 0?

Reply:
Yeah that definitely poses a problem. This is a limitation of the parsing library unfortunately. I think the best thing to do is change the type to a string, so that we can assert existence, and then parse it to `msg.ChainId` when we use it here: https://github.com/ChainSafe/ChainBridge/blob/7843f9d0d60f30b0f97f04c824c4bedc223411ce/cmd/chainbridge/main.go#L154